lib/exercism/comment.rb

```diff
@@ -36,4 +36,16 @@ def submission_user
   def mentions
     ExtractsMentionsFromMarkdown.extract(body)
   end
+
+  def qualifying?
+    submission_user != user
+  end
+
+  def increment_five_a_day
+    if FiveADayCount.exists?(:user_id => user.id)
+      FiveADayCount.where(:user_id => user.id).first.increment!(:total)
```

Comment:
There is only ever one record for in five a day for a single user, right? If so, there's no reason to say `first` here.